File: CloudDataProtection/CloudDataProtection.Services.OnboardingService/Dto/OnboardingResult.cs
using System;
using CloudDataProtection.Services.Onboarding.Entities;

namespace CloudDataProtection.Services.Onboarding.Dto
{
    public class OnboardingResult
    {
        public long Id { get; set; }
        
        public DateTime Created { get; set; }

        public OnboardingStatus Status { get; set; }
        
        public long UserId { get; set; }
        
        public GoogleLoginInfoResult LoginInfo { get; set; }
    }
}

File: cloud-data-protection/src/services/result/onboarding/onboardingResult.ts
import OnboardingStatus from "entities/onboardingStatus";
import GoogleLoginInfoResult from "services/input/onboarding/googleLoginInfoResult";

interface OnboardingResult {
    id: number;
    userId: number;
    created: Date;
    status: OnboardingStatus;
    loginInfo: GoogleLoginInfoResult;
}

export default OnboardingResult;